app/test.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+Unit tests for the herdbook.
+"""
+
+import unittest
+import requests
+
+import db
+
+HOST = "http://localhost:4200"
+
+class TestEndpoints(unittest.TestCase):
+    """
+    Checks that all endpoints are valid.
+    """
+
+    def test_main(self):
+        """
+        Checks that the main endpoint (/) is available.
+        """
+        self.assertEqual(
+            requests.get(HOST + '/').status_code,
+            200
+        )
+
+    def test_get_genebanks(self):
+        """
+        Checks that the genebanks endpoint (/api/genebanks) is available.
+        """
+        self.assertEqual(
+            requests.get(HOST + '/api/genebanks').status_code,
+            200
+        )
+
+    def test_get_individual(self):
+        """
+        Checks that the individuals endpoint (/api/individual/:id) is available.
+        """
+        self.assertEqual(
+            requests.get(HOST + '/api/individual/23').status_code,
+            200
+        )
+
+
+class TestDatabaseMapping(unittest.TestCase):
+    """
+    Checks that the database mapping is valid.
+    """
+    def setUp(self):
+        db.set_database("herdbook", "localhost", "5432", "herdbook", "insecure")
+
+    def test_genebank(self):
+        """
+        Checks that the genebank table exists and that all the fields in the
+        mapping exists in the database.
+        """
+        self.assertTrue(db.Genebank.table_exists())
+        db.Genebank.select().execute(db.DB_PROXY)
```

Comment:
Could we make sure that the models are actually bound to the given DB? I.e: `assert Class._meta.database is db.DB_PROXY`

Reply:
Is this relevant still?